style.css

```diff
@@ -0,0 +1,720 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+/* ------------------navbar section------------------- */
+.nav-container {
+  display: flex;
+  width: 1440px;
+  height: 72px;
+  background-color: #3e3c3c;
+  align-items: center;
+}
+
+.nav-container a {
+  text-decoration: none;
+  color: white;
+  font-family: 'Roboto', sans-serif;
+  padding-left: 20px;
+}
+
+.nav-container .items-menu {
+  display: flex;
+  align-items: center;
+  margin-left: 700px;
+  flex: 1;
+  height: 100%;
+  color: white;
+  text-decoration: none;
+  list-style: none;
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+  font-weight: 700px;
+}
+
+.nav-container .items-menu li a {
+  margin-right: 40px;
+  color: white;
+  list-style: none;
+  text-decoration: none;
+  font-size: 18px;
+  height: 100%;
+  padding-left: 10px;
+  padding-right: 10px;
+  transition: ease-in-out 5s;
+}
+
+.nav-container .items-menu li a:hover {
+  background-color: #615f5f;
+}
+
+.nav-container a .fa-bars {
+  display: none;
+}
+
+/* ---------------------------------------hero section------------------------------------ */
+.hero-section .hero-row {
+  display: flex;
+  background-color: #1c1a19;
+  width: 1440px;
+  height: 500px;
+}
+
+.hero-left {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-left: 30px;
+  margin-top: 100px;
+  flex: 0.2;
+  height: 100px;
+  transition: transform 5s;
+  order: 1;
+}
+
```

Comment:
[OPTIONAL] You did a great job justifying elements but also please make sure it is aligned with the text above.
<img width="366" alt="Screenshot 2023-01-16 at 9 03 36 PM" src="https://user-images.githubusercontent.com/25887253/212741966-faa682f2-f18a-4069-a7eb-895dfd6acef1.png">